internal/lib/lib.go

```diff
@@ -199,15 +199,12 @@ func GetHeaderRewritePolicy(vsName, localHost string) string {
 	return vsName + "--host-hdr-re-write" + "--" + localHost
 }
 
-func GetSniNodeName(ingName, namespace, secret, infrasetting string, sniHostName ...string) string {
+func GetSniNodeName(ingName, infrasetting, sniHostName string) string {
 	namePrefix := NamePrefix
 	if infrasetting != "" {
 		namePrefix += infrasetting + "-"
 	}
-	if len(sniHostName) > 0 {
-		return namePrefix + sniHostName[0]
-	}
-	return namePrefix + ingName + "-" + namespace + "-" + secret
+	return namePrefix + sniHostName
```

Comment:
does this change the naming convention of existing objects since it may have had the `secret` variable in the end?

Reply:
this is dead code for namespace sharding which is deprecated now.

Reply:
Isn't this used for hostname shard too .. from GetSniNodeName() ?

Reply:
yes, but the naming scheme where we use the `secret` was only used for namespace shard.

Reply:
Yes, Secret is passed as "". The secret can be removed.